code/game/objects/items/devices/PDA/cart.dm

```diff
@@ -656,6 +657,10 @@ Code:
 			loc:mode = 433
 			mode = 433
 
+		if("Supply Orders")
+			loc:mode =47

```

Comment:
>colons

Never ever, typecast that shit

And if you give me the "well everything else here does it" excuse

The nazis didn't get away from their trials with "b-but everyone else was gassing the jews"